tests/serverpod_test_server/test_integration/column_operations/column_string_test.dart

```diff
@@ -0,0 +1,166 @@
+import 'package:serverpod_test_server/test_util/test_serverpod.dart';
+import 'package:test/test.dart';
+
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_test_server/src/generated/protocol.dart';
+
+Future<void> _createTestDatabase(Session session) async {
+  var types = [
+    Types(aString: 'one'),
+    Types(aString: 'two'),
+    Types(aString: null),
+  ];
+
+  await Types.db.insert(session, types);
```

Comment:
NIT Suggestion: Create inlined to the insertion since we don't need a handle to the data types.

Reply:
(Might apply to the other migrated column tests as well)